server/build/webpack.js

```diff
@@ -280,6 +280,7 @@ export default async function createCompiler (dir, { dev = false, quiet = false,
       }
     },
     resolve: {
+      extensions: ['', '.js', '.json'],
```

Comment:
I think we can't have empty string here.
That's the root cause for this issue: https://travis-ci.org/zeit/next.js/jobs/216011309